src/lib/network_pool/network_pool_base.ml

```diff
@@ -45,7 +45,17 @@ end)
     network_pool
 
   (* Rebroadcast locally generated pool items every 5 minutes or every slot,
-     whichever is slower. Do so for half an hour before giving up. *)
+     whichever is slower. Do so for half an hour before giving up.
+
+     The goal here is to be resilient to short term network failures and
+     partitions. Note that with gossip we don't know anything about the state of
+     other peers' pools (we know if something made it into a block, but that can
+     take a long time and it's possible for things to be successfully received
+     but never used in a block), so in a healthy network all repetition is spam.
+     We need to balance reliability with efficiency. Exponential "backoff" would
+     be better, but it'd complicate the interface between this module and the
+     specific pool implementations.
```

Comment:
makes sense -- can we make it something like `max(3 blocks -- or if that's too hard from this module 10 slots, 30min)` so testnets don't suffer too much. We can easily go 4-5 slots in a row empty during these early staking testnets if a lot of the users' nodes aren't online (given we have 6min slots right now)

Reply:
or maybe Deepthi's parameters are better for this